src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -833,7 +844,8 @@ def load_arguments(self, _):
         c.argument('virtual_network_name', virtual_network_name_type, options_list=('--name', '-n'), id_part='name')
         c.argument('vnet_prefixes', nargs='+', help='Space-separated list of IP address prefixes for the VNet.', options_list=('--address-prefixes',), metavar='PREFIX')
         c.argument('dns_servers', nargs='+', help='Space-separated list of DNS server IP addresses.', metavar='IP')
-        c.argument('ddos_protection', arg_type=get_three_state_flag(), help='Enable DDoS protection for protected resources in the VNet.', min_api='2017-09-01')
+        c.argument('ddos_protection', arg_type=get_three_state_flag(), help=argparse.SUPPRESS, min_api='2017-09-01')
```

Comment:
I assume you will hide it from the command help, but still accept it? If yes, then in code you should emit out a warning telling people that they should not use it any more

Reply:
Actually, thanks for pointing this out--we need to still expose this flag.